DotnetViewComponents/Views/Shared/Components/SummaryList/Default.cshtml

```diff
@@ -0,0 +1,31 @@
+@using DotnetViewComponents.ViewModels
+@model SummaryListViewModel
+
+@{
+    bool hasActions = !String.IsNullOrWhiteSpace(@Model.SummaryListItem.First().ActionUrl);
+}
+
+<dl class="nhsuk-summary-list @(!Model.HasBorder ?  "nhsuk-summary-list--no-border" : "")">
+    @foreach (var listItem in Model.SummaryListItem)
+    {
+        <div class="nhsuk-summary-list__row">
+            <dt class="nhsuk-summary-list__key">
+                @listItem.Title
+            </dt>
+            <dd class="nhsuk-summary-list__value">
+                @listItem.Description
+            </dd>
+
+            @if (hasActions)
+            {
+                <dd class="nhsuk-summary-list__actions">
+
+                    <a href=@listItem.ActionUrl>
```

Comment:
Might it be more robust if we passed this `asp-controller=`, `asp-action=` and `asp-all-route-data=` parameters rather than relying on the dev to manually generate the URL? 

It would stop us from having to handle/work out different domain routes for different environments and/or relative paths.